modules/openapi-generator/src/main/resources/r/api_exception.mustache

```diff
@@ -32,15 +32,15 @@ ApiException <- R6::R6Class(
     initialize = function(status = NULL, reason = NULL, http_response = NULL) {
       if (!is.null(http_response)) {
         self$status <- http_response$status_code
-        errorMsg <- http_response$response
+        errorMsg <- http_response$ResponseAsText()
         if (is.null(errorMsg) || errorMsg == "") {
           errorMsg <- "Api exception encountered. No details given."
         }
         self$body <- errorMsg
         self$headers <- http_response$headers
         self$reason <- http_response$http_status_desc
         {{#errorObjectType}}
-        self$error_object <- {{errorObjectType}}$new()$fromJSONString(http_response$response)
+        self$error_object <- {{errorObjectType}}$new()$fromJSONString(http_response$ResponseAsText())
```

Comment:
These changes fix some evidently-broken R unit tests. The unit tests run cleanly for me locally now (replacing `"http://petstore.swagger.io/v2"` with `"http://localhost/v2"` in the API client due to limitations editing `/etc/hosts`).